docs/reference/remap/expressions/coalesce.cue

```diff
@@ -38,7 +38,7 @@ remap: expressions: coalesce: {
 		{
 			title: "Error coalescing"
 			source: #"""
-				parse_syslog("not syslog") ?? parse_apache_log("not apache") ?? "malformed"
+				parse_syslog("not syslog") ?? parse_common_log("not common") ?? "malformed"
```

Comment:
We don't have an `parse_apache_log` function yet (PR pending https://github.com/timberio/vector/pull/6608). Ref: #5369

Reply:
We do now!